app/assets/javascripts/discourse/app/routes/discovery.js

```diff
@@ -77,10 +77,11 @@ export default DiscourseRoute.extend(OpenComposer, {
 
   @action
   createTopic() {
+    const controller = this.controllerFor("discovery/topics");
     if (this.get("currentUser.has_topic_draft")) {
       this.openTopicDraft();
     } else {
-      this.openComposer(this.controllerFor("discovery/topics"));
+      this.openComposer(controller);
```

Comment:
The change in this file does not seem necessary.

Reply:
Ah yes this must've been accidentally added when I was testing things before. Thanks for catching that!